map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -687,6 +691,7 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
     override fun onDestroyView() {
         mapView.onDestroy()
         leaveTourDialog?.dismissAllowingStateLoss()
+        infoAdapter = null
```

Comment:
ah, but, um, the GoogleMap object can still use the object here...don't we need to call `mapView.setInfoWindowAdapter(null)`?

Reply:
I've done just that, in `MapViewModel::cleanup`. @caguilar187 just a head's-up: we will need you to help verify this code before long.